packages/now-php-bridge/config.js

```diff
@@ -0,0 +1,27 @@
+const DEFAULTS_MODE = 'server';
+const DEFAULTS_V = '7.3';
+
+const RUNTIME_NODE8 = 'nodejs8.10';
+const RUNTIME_NODE10 = 'nodejs10.x';
+
+function getMode(config) {
+  return !config || !config.mode ? DEFAULTS_MODE : config.mode;
+}
+
+function getVersion(config) {
+  return !config || !config.v ? DEFAULTS_V : config.v;
```

Comment:
```suggestion
  return !config || !config.php ? DEFAULTS_V : config.v;
```

Let's use `php` to define the version in case we need to version something else like `composer`.